sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -3,8 +3,10 @@
 ## 2.0.0-beta.2 (Unreleased)
```

Comment:
we have to discuss with Java Architects if we will go with major version bump

Reply:
Architects agreed with minor version bump.